src/gpu/intel/jit/conv/config.cpp

```diff
@@ -894,7 +894,7 @@ status_t init_tensor_layouts(
 }
 
 bool hw_ok(const hw_t &hw) {
-    if (hw < ngen::HW::Gen9) return false;
+    if (hw < ngen::HW::Gen12LP) return false;
```

Comment:
For consistency:

```suggestion
    if (hw < ngen::HW::XeLP) return false;
```

Gen12LP is mostly used in GEMM, while other code uses XeLP